run_from_source.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
This change should be reverted as is specific to your system, and we should remain 2.7 compatible